src/components/tutorial/Annotate.vue

```diff
@@ -19,55 +19,44 @@
 
             <h1>Label bounding boxes</h1>
 
-            <h4>Bottles</h4>
+            <h4>Known categories</h4>
+            <p>Group containing a few categories which are frequently observed. Together they make a big part of what is usually found on our sites.</p>
             <ul class="box">
-                <p>This tag relates to drink related items:</p>
+                <p><strong>Rigid, in one piece:</strong> subgroup containing all objects which somehow have a definite shape and hold in place. They wouldn’t change shape so much if put somewhere else (as opposed to, for example, a plastic bag).</p>
                 <li>
-                    drink packaging (plastic bottles, glass bottles, metal cans,
-                    bricks)
+                    Insulating material: blocks of material (sometimes floating in water), used for insulation e.g. foam and polystyrene 
                 </li>
                 <li>
-                    cleaning and chemical products stored in plastic bottles,
-                    glass bottles or metal cans
+                    Tire: self-explanatory e.g. truck tire, car tire, bike tire.
                 </li>
-                <li>
-                    drums &#60; 5 Liters
-                    <p>
-                        Characteristics: any color, any shape, any size (&#60; 5
-                        Litersl) also count cleaning and chemical products
-                    </p>
-                </li>
-                <p>
-                    <strong>Labeling tips</strong> Do not label small fragments
-                    of bottles, label each bottle separately even if they are
-                    nearby
-                </p>
                 <p>
-                    ⚠️ This tag do NOT include corks, cups, small flasks (&#60;
-                    500 ml)
                 </p>
-            </ul>
-            <h4>Fragments</h4>
-            <ul class="box">
-                <p>This tag relates to thin plastic fragments from:</p>
-                <li>plastic bags</li>
-                <li>tarps</li>
-                <li>clothes or textile</li>
+                <p><U>Liquid containers:</U> objects having a hollow shape for liquid containing, of various size and type.</p>
```

Comment:
```suggestion
                <p><u>Liquid containers:</u> objects having a hollow shape for liquid containing, of various size and type.</p>
```
On utilise plus volontiers du style pour

Reply:
On évite généralement d'utiliser `<u>` pour souligner un texte. On préfère ajouter du style avec un `text-decoration: underline`

Reply:
J'ai rajouté également un petit `<li>`, car les `<ul>` ne peuvent contenir que ce type de balise.